.github/workflows/source-and-docs-release.yml

```diff
@@ -80,20 +79,53 @@ jobs:
           path: |
             cpython/${{ inputs.cpython_release }}/src
 
+  build-docs:
+    runs-on: ubuntu-22.04
+    needs:
+      - verify-input
+
+    # Docs aren't built for alpha or beta releases.
+    if: ${{ !(contains(inputs.cpython_release, 'a') || contains(inputs.cpython_release, 'b')) }}
+
+    steps:
+      - name: "Checkout ${{ inputs.git_remote }}/cpython"
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          repository: "${{ inputs.git_remote }}/cpython"
+          ref: "v${{ inputs.cpython_release }}"
+
+      - name: "Setup Python"
+        uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
+        with:
+          python-version: 3.11
```

Comment:
We could bump this and the other to 3.12 at some point?

Reply:
Agreed, I can submit a PR for that too.